pulsar-broker/src/main/java/com/yahoo/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -969,7 +967,12 @@ public void updateRates(NamespaceStats nsStats, NamespaceBundleStats bundleStats
                 destStatsStream.writePair("msgRateRedeliver", subMsgRateRedeliver);
                 destStatsStream.writePair("type", subscription.getTypeString());
                 if (SubType.Shared.equals(subscription.getType())) {
-                    destStatsStream.writePair("unackedMessages", subUnackedMessages);
+                    if(subscription.getDispatcher() instanceof PersistentDispatcherMultipleConsumers) {
+                        PersistentDispatcherMultipleConsumers dispatcher = (PersistentDispatcherMultipleConsumers)subscription.getDispatcher();
+                        destStatsStream.writePair("blockedDispatcherOnUnackedMsgs",  dispatcher.isBlockedDispatcherOnUnackedMsgs());
```

Comment:
good catch. missed to update `blockedConsumerOnUnackedMsgs` for `getStatus()` api, pushed the [fix](https://github.com/yahoo/pulsar/pull/399/commits/1f291922e2210aa6b4a11c220d8dd37d3ade66b1)